ZeroPhone/ZPUI
# Update script

**Purpose**
make the upgrade of ZPUI from ZPUI as safe as possible.
See #34 for discussion

**Implementation**
- fixes import in `ui/test/*` files
- reformat `ui/test/*` files
- additions to `update.sh`
    - adds `set -e` to fail on **any** error encountered
    - adds actual branch name when pulling from git
    - ensure the script is run from the right `CWD`
    - adds dependency update/installation using `pip`
    - ensure creation of the install dir (in case the name changes)

** Issues**
The checkbox test fails for some reason. Needs to be fixed but not sure how to proceed yet. It's ignored for now (see [bpast-copy of the error stack](https://bpaste.net/show/69892fb5ecb0) )
Maybe the script should create the folder where personal data is stored as well (~/.phone/ ? )
